src/OnboardingSPA/steps/SiteGen/Migration/stylesheet.scss

```
@@ -0,0 +1,56 @@
+.nfd-onboarding-step--site-gen__migration {
```

Shall we this at every `--` and `__`? So this would become
```suggestion
.nfd-onboarding-step {
    --site-gen {
        __migration
    ....
    }
}
```